game-core/src/main/java/games/strategy/triplea/ui/BattleDisplay.java

```diff
@@ -875,15 +876,15 @@ void refresh() {
       }
       final List<Unit> units = new ArrayList<>(this.units);
       DiceRoll.sortByStrength(units, !attack);
-      final Map<Unit, TotalPowerAndTotalRolls> unitPowerAndRollsMap;
+      final TotalPowerAndTotalRolls unitPowerAndRollsMap;
       final boolean isAirPreBattleOrPreRaid = battleType.isAirBattle();
       if (isAirPreBattleOrPreRaid) {
-        unitPowerAndRollsMap = Map.of();
+        unitPowerAndRollsMap = null;
```

Comment:
In a later PR, I plan to make a new `TotalPowerAndTotalRolls` implementation for air attack/defense. That will be used here.